src/Compilers/VisualBasic/Test/Emit/Attributes/AttributeTests_CallerArgumentExpression.vb

```diff
@@ -52,14 +52,50 @@ End Module
 "
 
             Dim compilation = CreateCompilation(source, targetFramework:=TargetFramework.NetCoreApp, references:={Net451.MicrosoftVisualBasic}, options:=TestOptions.ReleaseExe, parseOptions:=TestOptions.RegularLatest)
-            CompileAndVerify(compilation, expectedOutput:="123").VerifyDiagnostics()
-            ' PROTOTYPE(caller-expr): Confirm whether this should be case-sensitive or case-insensitive.
-            '            compilation.AssertTheseDiagnostics(
-            '<expected><![CDATA[
-            'BC42505: The CallerArgumentExpressionAttribute applied to parameter 'arg' will have no effect. It is applied with an invalid parameter name.
-            '    Sub Log(p As Integer, <CallerArgumentExpression(P)> Optional arg As String = "<default-arg>")
-            '                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~
-            ']]></expected>)
+            CompileAndVerify(compilation, expectedOutput:="123").VerifyDiagnostics().VerifyTypeIL("Program", "
```

Comment:
Consider adding a test that defines an incorrectly-cased public method in VB and use it from a C# program.